libraries-data-mariadb4j/src/test/java/com/baeldung/data/mariadb4j/JavaLiveTest.java

```diff
@@ -26,10 +28,12 @@ void whenStartingADatabase_thenTheDatabaseIsUsable() throws Exception {
 
     @Test
     void whenStartingADatabaseWithConfiguration_thenTheDatabaseIsUsable() throws Exception {
+        Path tempDir = Files.createTempDirectory("mariadb");
```

Comment:
This temporary directory isn't cleaned up after the build - use something like `@TempDir` instead https://www.baeldung.com/junit-5-temporary-directory

Reply:
I didn't know about that one. Updated. :)